README.md

```diff
@@ -49,7 +49,7 @@ Collection of npm modules created by Wire's web team:
 
 ```bash
 ## Download dependencies and link packages
-yarn && yarn boot
+yarn
```

Comment:
does yarn2 provides the exact same feature as lerna bootstrap? Why did we need the bootstrap before?

Reply:
as far as i understand, lerna bootstrap is from ancient times when npm had no workspaces and yarn didn't exist.
Yarn2 has a very good workspace integration that takes care of the things lerna bootstrap was supposed to do.